Makefile

```diff
@@ -243,7 +265,12 @@ install-strip:
 	cd $(LIB_INSTALL_PATH) && $(STRIP_EXEC) $(DEP_TO_LIB)
 	cd $(LUA_INSTALL_CPATH) && $(STRIP_EXEC) *.so
 
-install: install-dep install-emulator install-strip $(INSTALL_PLAT)
+install: install-dep install-emulator install-strip $(INSTALL_PLAT) install-tests
+debian: install-dep install-emulator install-strip $(INSTALL_PLAT)
```

Comment:
I agree with the suggestion about renaming to `make deb` or `make deb-package`. So we can repurpose this to make packages for any .deb based distro (Ubuntu, Debian or other).